frontend/src/container/NotificationWall.jsx

```diff
@@ -161,6 +162,20 @@ export class NotificationWall extends React.Component {
     return { icon, text, url }
   }
 
+  handleClickMarkAllAsRead = async () => {
+    const { props } = this
+
+    const fetchAllPutNotificationAsRead = await props.dispatch(putAllNotificationAsRead(props.user.userId))
+    switch (fetchAllPutNotificationAsRead.status) {
+      case 204: {
+        props.notificationPage.list.forEach(notification => props.dispatch(updateNotification({ ...notification, read: true })))
```

Comment:
I'd find this more readable:
```suggestion
        for (const notification of props.notificationPage.list) {
            props.dispatch(updateNotification({ ...notification, read: true }))
        }
```
I think we really should avoid forEach when `for…of` works. I don't see any advantage of using forEach in these cases (but I know this is what we currently do in the frontend). `forEach` is no more functional than `for` and was great when `for..of` was still not supported by browsers.

Invoking @Skylsmoi who may not agree with me here.

Reply:
For me `forEach` is more readable, but I can wait to see @Skylsmoi like a casting vote.

Reply:
If you find it more readable, keep it as is ;-)